### ISSUE TITLE
How to use the finetuned model

### ISSUE THREAD

**Issue by aaahuia** [X-PLUG/mPLUG-Owl#31]
I run train_it.sh on my own data
I got the file as below：

optimizer.pt       rng_state_0.pth  rng_state_2.pth  rng_state_4.pth  rng_state_6.pth  scheduler.pt        training_args.bin
pytorch_model.bin  rng_state_1.pth  rng_state_3.pth  rng_state_5.pth  rng_state_7.pth  trainer_state.json

How to use this model

**Comment by lambertjf** [comment]
Also curious about this. I tried what was suggested in another issue, downloading the model again from HuggingFace and replacing the pytorch_model.bin, but this produced a warning about some weights not being used, and then the output was nonsensical random tokens. I'm thinking that something else needs to be done after LoRA training for it to work correctly

**Comment by FuxiaoLiu** [comment]
> I run train_it.sh on my own data I got the file as below：
> 
> optimizer.pt rng_state_0.pth rng_state_2.pth rng_state_4.pth rng_state_6.pth scheduler.pt training_args.bin pytorch_model.bin rng_state_1.pth rng_state_3.pth rng_state_5.pth rng_state_7.pth trainer_state.json
> 
> How to use this model

Do you solve this problem?

**Comment by MAGAer13** [comment]
> > I run train_it.sh on my own data I got the file as below：
> > optimizer.pt rng_state_0.pth rng_state_2.pth rng_state_4.pth rng_state_6.pth scheduler.pt training_args.bin pytorch_model.bin rng_state_1.pth rng_state_3.pth rng_state_5.pth rng_state_7.pth trainer_state.json
> > How to use this model
> 
> Do you solve this problem?

Hi, please refer to this issue. https://github.com/X-PLUG/mPLUG-Owl/issues/87#issuecomment-1608558909
